docs/basics/app-anatomy.md

```diff
@@ -149,7 +149,9 @@ The [`handler`](../building-modules/handler.md) refers to the part of the module
 The `handler` of a module is generally defined in a file called `handler.go` and consists of:
 
 - A **switch function** `NewHandler` to route the message to the appropriate `handler` function. This function returns a `handler` function, and is registered in the [`AppModule`](#application-module-interface) to be used in the application's module manager to initialize the [application's router](../core/baseapp.md#routing). Next is an example of such a switch from the [nameservice tutorial](https://github.com/cosmos/sdk-tutorials/tree/master/nameservice)
-    +++ https://github.com/cosmos/sdk-tutorials/blob/master/nameservice/x/nameservice/handler.go#L12-L26
+
++++ https://github.com/cosmos/sdk-tutorials/blob/86a27321cf89cc637581762e953d0c07f8c78ece/nameservice/x/nameservice/internal/keeper/querier.go#L19-L32
```

Comment:
Eg this is a different commit

Reply:
Yes, that's because I'm using the updated URL from `master` — https://github.com/cosmos/cosmos-sdk/blame/master/docs/basics/app-anatomy.md#L209